gems/quilt_rails/test/quilt_rails/controllers/routing_test.rb

```diff
@@ -32,7 +32,7 @@ class UiController < ActionController::Base; end
       boot_dummy
 
       Rails.application.routes.draw do
-        mount(Quilt::Engine, at: '/quilt')
+        mount(::Quilt::Engine, at: '/quilt')
```

Comment:
what does the prefix `::` mean?

Reply:
`QuiltRails::Quilt` gets defined by a controller test, so `::` makes sure the toplevel `Quilt` module gets referenced.